MetaCocktailsSwiftData/Model/Cocktails/Classics/LastWord.swift

```diff
@@ -10,7 +10,7 @@ import SwiftUI
 var lastWord = Cocktail(cocktailName: "Last Word",
                         glasswareType: .stemmedGlassware,
                         ice: nil,
-                        author: "Frank Fogarty or the Detroit Athletic club",
+                        author: ["Frank Fogarty or the Detroit Athletic club"],
```

Comment:
Was Frank Fogarty the bartender at the Detroit Athletic club who came up with it?